benchmarking/benchmark.py

```diff
@@ -314,39 +315,39 @@ def run_experiments(config):
             logger.info('running experiment: {}'.format(current_experiment))
             if repetition != 1:
                 logger.info('\trepetition {} out of {}'.format(rep + 1, repetition))
-            result = run_single_experiment(server, config, threshold, sizes, current_experiment)
+            result = run_single_experiment(config, threshold, sizes, current_experiment)
             results['experiments'].append(result)
 
     return results
 
 
-def run_single_experiment(server, config, threshold, sizes, experiment):
+def run_single_experiment(config, threshold, sizes, experiment):
     result = {}
     credentials = {}
     run = {}
     logger.info("Starting time: {}".format(time.asctime()))
     nb_parties = len(sizes)
     try:
-        credentials = rest_client.project_create(server, config['schema'], 'groups',
+        credentials = rest_client.project_create(config['schema'], 'groups',
                                                  "benchy_{}".format(experiment), parties=nb_parties)
         # upload clks
         upload_binary_clks(config, sizes, credentials)
         # create run
         project_id = credentials['project_id']
         result['project_id'] = project_id
-        run = rest_client.run_create(server, project_id, credentials['result_token'],
+        run = rest_client.run_create(project_id, credentials['result_token'],
                                      threshold,
                                      "{}_{}".format(experiment, threshold))
         # wait for result
         run_id = run['run_id']
         result['run_id'] = run_id
         logger.info(f'waiting for run {run_id} from the project {project_id} to finish')
-        status = rest_client.wait_for_run(server, project_id, run_id,
-                                          credentials['result_token'], timeout=config['timeout'])
+        status = rest_client.wait_for_run(project_id, run_id,
+                                          credentials['result_token'], timeout=config['timeout'], update_period=5)
```

Comment:
will that lead to a new line in the logs every 5 seconds? what about those huge benchmarks? How big's the log file going to be?

Reply:
Not on the benchmark container side - we just wait until the whole thing is done (perhaps we should be outputting progress but that is another question).

On the server side we would log the request in both nginx and the flask app.